components/ws-manager-bridge/src/bridge-controller.ts

```diff
@@ -64,7 +64,7 @@ export class BridgeController {
     protected async reconcile() {
         return this.reconcileQueue.enqueue(async () => {
             const allClusters = await this.getAllWorkspaceClusters();
-            log.info("reconciling clusters...", { allClusters: Array.from(allClusters.values()) });
+            log.info("reconciling clusters...", { allClusters: Array.from(allClusters.keys()) });
```

Comment:
[nit] Logs should start with first letter uppercase

Reply:
I'm happy to help setup a guideline for this. As long as we don't have that I think it's out of scope here. :slightly_smiling_face: